### DISCUSSION TITLE
input data

### DISCUSSION THREAD

**Discussion by fujch7** [ablab/spades#768] [Q&A] [answered]
Hi，
   Is there any differences between these 2 cmd, using _--pe1-1 --pe1-2_ or _-1 -2_. 
   T4.R1.fq.gz and T4.R2.fq.gz were paired-end.

spades.py --meta -t 32 -m 1536 --pe1-1 T4.R1.fq.gz --pe1-2 T4.R2.fq.gz -o spadesAssembly/

spades.py --meta -t 32 -m 1536 -1 T4.R1.fq.gz -2 T4.R2.fq.gz -o spadesAssembly/

**Comment by asl** [comment, marked as answer]
As indicated in SPAdes manual (https://cab.spbu.ru/files/release3.15.2/manual.html#sec3.1), these are fully equivalent
